### ISSUE TITLE
[SFS] The contact person and report/autors should use the same infos

### ISSUE THREAD

**Issue by Ludee** [OpenEnergyPlatform/oeplatform#1512]
### Factsheet
Study / Scenario

### Field
S6: Contact person
R6: Authors

## Ideas of solution
Make the "person" available for both fields

**Comment by jh-RLI** [comment]
No, this has not yet been implemented. When searching for “Ludwig” in the contact person field, an entry is found, but not when searching in the authors field. I can fix this as soon as I have time / I'm happy to help anyone who wants to fix this.
